docs/src/manual.md

```diff
@@ -49,6 +49,60 @@ used instead.
 
 An intercept is included in any GLM by default.
 
+## Categorical variables
+
+Categorical variables will be dummy coded by default if they are non-numeric or if they are [`CategoricalVector`s](https://juliadata.github.io/CategoricalArrays.jl/stable/) within a TableTraits-supporting table (DataFrames, JuliaDB, ColumnTable, etc). Alternatively, you can pass an explicit [contrasts](https://juliastats.github.io/StatsModels.jl/latest/contrasts/) argument if you would like a different contrast coding system or if you are not using DataFrames.
+
+The response (dependent) variable may not be categorical.
+
+Using a `CategoricalVector` constructed with `categorical`:
+
+```jldoctest
+julia> using DataFrames, GLM
+
+julia> data = DataFrame(y = rand(100), x = categorical(repeat([1, 2, 3, 4], 25)));
+
+julia> # It is also possible to convert an existing column to be categorical with categorical!(data, :x)
+
+julia> lm(@formula(y ~ x), data)
+StatsModels.TableRegressionModel{LinearModel{GLM.LmResp{Array{Float64,1}},GLM.DensePredChol{Float64, LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}
+
+y ~ 1 + x
+
+Coefficients:
+──────────────────────────────────────────────────────────────────────────────
+               Estimate  Std. Error    t value  Pr(>|t|)  Lower 95%  Upper 95%
+──────────────────────────────────────────────────────────────────────────────
+(Intercept)   0.561258    0.0612945   9.15675     <1e-14   0.439589  0.682927
+x: 2         -0.121297    0.0866835  -1.39931     0.1649  -0.293363  0.0507681
+x: 3         -0.109711    0.0866835  -1.26565     0.2087  -0.281776  0.0623548
+x: 4          0.0490837   0.0866835   0.566241    0.5726  -0.122982  0.221149
+──────────────────────────────────────────────────────────────────────────────
+```
+
+Using [`contrasts`](https://juliastats.github.io/StatsModels.jl/latest/contrasts/):
+
+```jldoctest
+julia> using DataFrames, GLM
+
+julia> data = DataFrame(y = rand(100), x = repeat([1, 2, 3, 4], 25));
```

Comment:
Give a name to this block and the previous one so that you can reuse `data` from there.

Reply:
I don't think I should do this because the `data` variables used are similar but not identical.